### ISSUE TITLE
ffmpeg

### ISSUE THREAD

**Issue by mhsmith** [chaquo/chaquopy#143]
This is required by number of packages which load video and audio.

**Comment by mhsmith** [comment]
Some packages expect to be able to call ffmpeg as an executable, in which case #605 would be an issue.

**Comment by puelon** [comment]
Hi, it would be great if we could use FFMPEG with chaquopy. Android is kinda of restricted with USB cameras and there is like 1 or 2 libraries that work with USB cameras. Currently, after a long time trying I was able to retrieve frames from my camera but I need to use sockets to stream my images, would be great if it was possible to use it with ffmpeg

**Comment by muizzzafar11** [comment]
any update on this issue?

**Comment by mhsmith** [comment]
No, but the package build tool is now open-source, so if you'd like to try building this package yourself, follow the instructions [here](https://github.com/chaquo/build-wheel). And if you're successful, please make a pull request so we can add it to the public repository.

As mentioned above, anything that requires calling ffmpeg as an *executable* will not work on API level 29 or later. So it's not worth making any effort to build it unless you intend to use it as a library.
